### ISSUE TITLE
Can it support custom download paths at runtime (needed for offline environments)?

### ISSUE THREAD

**Issue by ColinZeb** [clowd/Clowd.Squirrel#174]
For example, placing the runtime in the 'runtime' folder on the update website

**Comment by KennyTK** [comment]
Yes. 

Modified from README.md:

`
string UPDATEPATH = "https://the.place/you-host/updates";

private static async Task UpdateMyApp()
{
  if(offline = true)
  {
    UPDATEPATH = "\\server\my\offline\path\"
  }

   using var mgr = new UpdateManager(UPDATEPATH);
   var newVersion = await mgr.UpdateApp();
   
   // optionally restart the app automatically, or ask the user if/when they want to restart
   if (newVersion != null) {
      UpdateManager.RestartApp();
   }
}`

Use UpdateManager to specify different paths.

**Comment by ColinZeb** [comment]
@KennyTK My issue is whether it is possible to customize the runtime path during execution. For example, if I deploy an update website locally or on an intranet server, can the client detect the absence of a runtime and download it from the intranet instead of from the Microsoft website?

**Comment by KennyTK** [comment]
@ColinZeb I think there may be a conflation between runtime, framework, and the purpose of Clowd. When you are referring to runtime, do you mean [dot net framework](https://dotnet.microsoft.com/en-us/download/dotnet-framework)?

**Comment by ColinZeb** [comment]
@KennyTK Yes, I need to be able to install the .net framework automatically when installing in a brand new environment, not from the Internet, but from an update site.

**Comment by KennyTK** [comment]
This method gets the url for downloading runtime.

[Squirrel::Runtimes::DotnetInfo::GetDotNetDownloadUrl](https://github.com/clowd/Clowd.Squirrel/blob/289867fbba9222e80d0f20b2d37ddbe2fa32a13e/src/Squirrel/RuntimeInfo.cs#L428C48-L428C48)
